sphinx/api/built-in/rose_arch.rst

```diff
@@ -164,71 +264,119 @@ Configuration
 
       .. rose:conf:: compress=pax|tar|pax.gz|tar.gz|tgz|gz
 
-         If specified, compress source files to the given scheme before
-         sending them to the archive. If not specified, the compress
-         scheme is automatically determined by the file extension of
-         the target, if it matches one of the allowed values. For the
-         ``pax|tar`` scheme, the sources will be placed in a TAR archive
-         before being sent to the target. For the ``pax.gz|tar.gz|tgz``
-         scheme, the sources will be placed in a TAR-GZIP file before
-         being sent to the target. For the ``gz`` scheme, each source
-         file will be compressed by GZIP before being sent to the target.
+         If specified, compress source files scheme before sending them to the
+         archive. If not set Rose Arch will attempt to set a compression scheme
+         if the file extension of the target implies compression: For
+         example, setting target as ``[arch:example.tar]`` is the same as
+         setting ``compress=tar``.
+
+         Each compression scheme works slightly differently:
+
+         +------------------+-----------------------------------------------+
+         |Compression Scheme|Behaviour                                      |
```

Comment:
A work of beauty, nice.

For future reference drawing table borders in RST can be a bit of a pain so there are a couple of directives to make life easier:
* http://docutils.sourceforge.net/docs/ref/rst/directives.html#csv-table
* http://docutils.sourceforge.net/docs/ref/rst/directives.html#list-table
We already use `list-table` elsewhere in the docs, it can also be used with the `grid-table` extension https://github.com/cylc/cylc-sphinx-extensions/tree/master/cylc/sphinx_ext/grid_table.

Reply:
Yes, but your version make more sense.